src/collections/_documentation/development/sdk-dev/event-payloads/debugmeta.md

```diff
@@ -85,6 +85,21 @@ Attributes:
 : _Optional_. The absolute path to the dynamic library or executable. This helps
   to locate the file if it is missing on Sentry.
 
+`image_vmaddr`:
+
+: _Optional_: Preferred load address of the image in virtual memory, as declared
+  in the headers of the image. When loading an image, the operating system may
+  still choose to place it at a different address. 
+  
+  If this value is non-zero, all symbols and addresses declared in the native
+  image start at this address, rather than `0`. In contrast, Sentry deals with
```

Comment:
By contrast 

(In contrast is typically followed by "with" or "to"; by contrast is typically followed by the subject of the contrast :) )

Reply:
Thanks, also updated other parts in docs where this was used.

cc @MimiDumpling